README.md

```diff
@@ -2,16 +2,20 @@
 
 # nix-darwin
 
-![Test](https://github.com/LnL7/nix-darwin/workflows/Test/badge.svg)
+[![Test](https://github.com/LnL7/nix-darwin/actions/workflows/test.yml/badge.svg)](https://github.com/LnL7/nix-darwin/actions/workflows/test.yml)
```

Comment:
I suspect this badge does not really add any value. (But people love them, so…)

Reply:
The old badge added negative(?) value because it was always green even when CI failed 😆